CMakeLists.txt

```diff
@@ -118,10 +115,6 @@ install(FILES
     sketcherMinimizerStretchInteraction.h
     DESTINATION include/coordgen)
 
-install(FILES
-    templates.mae
-    DESTINATION share/coordgen)
-
```

Comment:
We still need this if we enable `COORDGEN_USE_MAEPARSER`, right?

Reply:
no - the templates are always loaded from the library; "templates.mae" is never used. COORDGEN_USE_MAEPARSER allows override using "user_templates.mae".

Reply:
Nevermind. GitHub's code reviews always get me confused. I can't get used to this formatting.